src/main/resources/Chunks.json

```diff
@@ -1,15 +1,29 @@
-{
-  "Chunk":{
-    "Chunk Number": 1,
-    "Chunk Name": "",
-    "Mobs": [
-      {"Name": "", "Level":1}
-      ],
-    "Skilling Resources":[],
-    "Item Spawns": [""],
-    "Processing Tools": [""],
-    "Shops":[],
-    "Other Notes":""
-  },
-  
-}
+[
+  {
+    "chunkNumber": 1,
+    "chunkName": "1",
+    "mobs": [
+      {
+        "name": "",
+        "combatLvl": 2,
+        "drops":[""]
+      }
+    ],
+    "skillingNodes": [],
+    "groundItems": [
+      ""
+    ],
+    "processingTools": [
+      ""
+    ],
+    "shops": [
+      {
+        "name": "Name",
+        "chunkLocation": [2000],
+        "stock":[""],
```

Comment:
Again, if we have a db of shops, no need to add stock to 2 json files.

Reply:
Solved